### ISSUE TITLE
Error loading Resources from wallet

### ISSUE THREAD

**Issue by jkdale** [greymass/anchor#887]
I imported my keys into Anchor 1.0.2 on macOS. When I try to view resources for my different wallets, I see an error

```
TypeError: Cannot read property 'split' of null
    at Function.mapToProps (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:9:1554226)
    at o (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:162044)
    at h (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:163321)
    at file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:163393
    at Object.run (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:159824)
    at i.l.onStateChange (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:160300)
    at Object.notify (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:157143)
    at e.t.notifyNestedSubs (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:156785)
    at i.l.onStateChange (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:160442)
    at Object.notify (file:///Applications/Anchor%20Wallet.app/Contents/Resources/app.asar/dist/renderer.main.prod.js:80:157143)

**Comment by aaroncox** [comment]
Thanks for the heads up - I think we have this fixed in the next build (1.0.3), but we'll check in once it's released (soon) to make sure it's working.

**Comment by Soleone** [comment]
I got that error as well. Works for a few seconds and eventually displays that error:

![image](https://user-images.githubusercontent.com/2923/82624669-29248f80-9bb1-11ea-89a5-cccb84b45b81.png)

**Comment by aaroncox** [comment]
This should be resolved in 1.0.3 now 👍 

Feel free to reopen if you experience the issue again.
